hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -212,42 +271,39 @@ prettyEvaluationReport (Map.toList -> xs) =
   prettyResult =
     either (T.replace "\n" " " . show) show
 
--- NOTE: Uses 'testPolicyId' for the datum.
-genAbortableOutputs :: [Party] -> Gen ([(TxIn, TxOut CtxUTxO)], [(TxIn, TxOut CtxUTxO, UTxO)])
-genAbortableOutputs parties =
```

Comment:
Why did you touch this function? (It's a notorious one, which should actually be rather deleted than touched)